addons/mail/static/src/new/core/messaging.js

```diff
@@ -715,6 +709,34 @@ export class Messaging {
         });
     }
 
+    async searchPartners(searchStr = "", limit = 10) {
+        let partners = [];
+        const searchTerm = cleanTerm(searchStr);
+        const odoobotId = this.state.partnerRoot.id;
+        for (const id in this.state.partners) {
+            const partner = this.state.partners[id];
+            // todo: need to filter out non-user partners (there was a user key)
+            if (
+                partner.id !== odoobotId &&
```

Comment:
I believe odoobot should be accepted here, or at least it wasn't specifically removed in old code.

Reply:
i think it was removed in a different way. maybe because it is not an active user?

Reply:
Using active for odoobot is not reliable as each DB has its own config in this regard (default is not active with demo data but it's active on our prod for example).